tests/test_bdist_pex.py

```diff
@@ -59,12 +45,12 @@ def do_something():
     """)
 
   with temporary_content({'setup.py': setup_py, 'my_app.py': my_app}) as project_dir:
-    my_app_pex = bdist_pex_installer(project_dir).bdist()
-    process = subprocess.Popen([my_app_pex], stdout=subprocess.PIPE)
-    stdout, _ = process.communicate()
-    assert '{pex_root}' not in os.listdir(project_dir)
-    assert 0 == process.returncode
-    assert stdout == b'hello world!\n'
+    with bdist_pex(project_dir) as my_app_pex:
+      process = subprocess.Popen([my_app_pex], stdout=subprocess.PIPE)
+      stdout, _ = process.communicate()
+      assert '{pex_root}' not in os.listdir(project_dir)
+      assert 0 == process.returncode
```

Comment:
Nit that was there before you. Should be `assert process.returncode == 0`. Feel free to ignore.

Reply:
Why should it be that way? For xunit, the formatting of failure messages has expected as the left arg, actual the right. This presumably just keeps that xunit convention. FWICT though ot does not matter since pytest failure messages for `assert`-style tests assume nothing about which slot is expected and which is actual. See example messages here: https://docs.pytest.org/en/latest/assert.html

Reply:
In fact - even python unittest messages do not make this distinction. Maybe only junit does?

Reply:
It is a long-standing tradition that expected goes on the left, actual on the right. Even if it doesn't matter in some context, might as well be consistent.

Reply:
I stand corrected. Python unittest used to have a suggested ordering, but changed it a couple years ago. See https://stackoverflow.com/questions/2404978/why-are-assertequals-parameters-in-the-order-expected-actual#2405193. 

Before it became ambiguous, they indeed used order expected, actual.